index.js

```diff
@@ -0,0 +1,53 @@
+let input = document.getElementById('operation-input');
```

Comment:
Please use `const` if you will not change the value

Reply:
ok